### ISSUE TITLE
Advertise events on large television near entrance

### ISSUE THREAD

**Issue by jackie1050** [DoESLiverpool/somebody-should#903]
Display details of events / important issues on the the large television (when it isn't being used for VR).

Will need some method to collect events etc., possibly by including a code in Twitter,  Google Groups messages/link to Calendar..

Also include photos from Instagram.

Hopefully a relatively automatic way to give community members another way to see messages they may have missed.

**Comment by johnmckerrell** [comment]
This made me think of something I thought I might have added as an issue a while ago but I'm not seeing it. I wondered about an automated presentation that we could show at the beginning of events in our space, it would pull in the events calendar and have pre-prepared slides for each event which we'd maybe fill with the date of the event that month or the topic or things like that which we'd pull from... somewhere, most likely the calendar. I would think the way to collect events would be our calendar.

**Comment by MatthewCroughan** [comment]
It'd be nice if there existed some software to do this from a github repo. I'm pretty sure there exists some software that would do this from some directory, so I could set up `pushd` scripts on a raspberry pi to allow all of this to be managed from github, making everything as simple as pushing and pulling, and less susceptible to accidental deletions, conflicts, and more as a local network share would be.

**Comment by DoESsean** [comment]
I've long been a proponent of using that and any other) tv to advertise the
people in the space too - monthly desk/workshop membership would get you an
ad that would run as part of this rotation, so that people get a better
idea of who is in the space and what they do.

**Comment by MatthewCroughan** [comment]
Started on this tonight, should be finished before I leave in an hour. Using Dakboard.com as a base, but will likely recreate the functionality in future. Looks pretty good for now though. Does what is outlined. 
![image](https://user-images.githubusercontent.com/26458780/46189377-4030b280-c2e6-11e8-8ceb-e0536de6ef4f.png)

**Comment by MatthewCroughan** [comment]
@ajlennon Worth noting, I borrowed your HDMI cable and a disconnected no-sd rpi from your desk to do this, so if you need it feel free to return it to its rightful location. I just needed to get this set up out of the way.

**Comment by ajlennon** [comment]
Thanks for letting me know :)

**Comment by MatthewCroughan** [comment]
@ajlennon, your pi isn't being used for the time being, so if you need it, it should be around behind the tv. I asked to borrow a pi 3 from John, will embed this calendar later using some awesome resin.io webkit stuff I found. Will share details as I discover them.

**Comment by ajlennon** [comment]
Hey resin.io! Nice! You might have already done this but can you talk to the Directors about whether you can create a DoES account on Resin.io to put all this stuff. :)

**Comment by DoESsean** [comment]
Is this something that we had a workable solution for? It would be good to have up and running.

**Comment by MatthewCroughan** [comment]
@DoESsean As far as I'm aware we set up the calendar on the TV by the entrance door, which has since died and @johnmckerrell has been working on. The calendar I set up using dakboard is still functional and we can display that if we'd like in the meantime.
